roles/openshift_logging_kibana/defaults/main.yml

```diff
@@ -41,3 +41,24 @@ openshift_logging_kibana_key: ""
 #The absolute path on the control node to the CA file to use
 #for the public facing kibana certs
 openshift_logging_kibana_ca: ""
+
+openshift_logging_kibana_ops_nodeselector: {}
```

Comment:
The goal of this role was to not have any knowledge of ops, things should be passed in from the logging role to it and specify that this would be an 'ops' deployment.